.github/workflows/clp-core-build.yaml

```
@@ -186,6 +285,170 @@ jobs:
             --build-dir /mnt/repo/components/core/build
             --num-jobs $(getconf _NPROCESSORS_ONLN)
 
+  manylinux_2_28-aarch64-binaries:
```

actionlint (1.7.7)

job "manylinux_2_28-aarch64-binaries" needs job "manylinux_2_28-aarch64-deps-image" which does not exist in this workflow

(job-needs)